src/main/webapp/script.js

```diff
@@ -60,22 +60,38 @@ function userLogin() {
 function loadGrocery() {
     // fetches json list of groceries
     fetch('/grocery-list').then(response => response.json()).then((groceries) => {
-        const groceryListElement = document.getElementById('grocery-list-container');
-      
+    const groceryListElement = document.getElementById('grocery-list-container');  
     groceries.forEach((grocery) => {
         groceryListElement.appendChild(createGroceryElement(grocery));
     })
   });
```

Comment:
Indent lines 63-66 and line up 67 with beginning of fetch

Reply:
solve